test/e2e/v1beta1/run-e2e-experiment.go

```diff
@@ -270,8 +270,13 @@ func verifyExperimentResults(kclient katibclient.Client, exp *experimentsv1beta1
 			*exp.Spec.MaxTrialCount, exp.Status.TrialsSucceeded)
 	}
 
+	trialsSucceeded := exp.Status.TrialsSucceeded
```

Comment:
I think we should name this var similar as `trialsCompleted` to be more precise.

Reply:
SGTM.

Reply:
Done.